game-core/src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -293,14 +293,6 @@ public void removeDataChangeListener(final GameDataChangeListener listener) {
     dataChangeListeners.remove(listener);
   }
 
-  public void addGameMapListener(final GameMapListener listener) {
-    gameMapListeners.add(listener);
-  }
-
-  public void removeGameMapListener(final GameMapListener listener) {
-    gameMapListeners.remove(listener);
-  }
-
```

Comment:
Does this mean there are no listeners at all? 
In this case all operations on gameMapListeners could be removed.
We need to keep the variable for backwards compatibility though.

Reply:
Following up here, variable access is not encapsulated by just these accessor methods. I tracked down a few more usages to:
```
  void notifyMapDataChanged() {
    gameMapListeners.forEach(GameMapListener::gameMapDataChanged);
  }

  public void clearAllListeners() {
    dataChangeListeners.clear();
    territoryListeners.clear();
    gameMapListeners.clear();
    if (resourceLoader != null) {
      resourceLoader.close();
      resourceLoader = null;
    }
  }
```

I think that probably hints why the methods are unused.

Reply:
Actually.. wait.. I think you're  right here  @RoiEXLab; taking a second look at this. :thinking: